pdart/pipeline/CopyPrimaryFiles.py

```diff
@@ -78,7 +78,9 @@ def _copy_fits_files(
                 instrument_name = hst_filename.instrument_name()
```

Comment:
BTW, I had not realize that Eric's "fs" code was still in use (see above). It was a mechanism for having versions of the file system so that we could more easily handle updates. We ultimately decided not to do it this way, so we should at some point strip out this complexity. However, we can save that task for another day.